frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx

```diff
@@ -100,60 +124,225 @@ export default function WorkspaceBlocked(): JSX.Element {
 		});
 	};
 
-	return (
-		<>
-			<FullScreenHeader overrideRoute={ROUTES.WORKSPACE_LOCKED} />
-
-			<Card className="workspace-locked-container">
-				{isLoadingLicenseData || !licensesData?.payload?.workSpaceBlock ? (
-					<Skeleton />
-				) : (
-					<>
-						<LockOutlined style={{ fontSize: '36px', color: '#08c' }} />
-						<Typography.Title level={4}> Workspace Locked </Typography.Title>
-						<Typography.Paragraph className="workpace-locked-details">
-							You have been locked out of your workspace because your trial ended
-							without an upgrade to a paid plan. Your data will continue to be ingested
-							till{' '}
-							{getFormattedDate(licensesData?.payload?.gracePeriodEnd || Date.now())} ,
-							at which point we will drop all the ingested data and terminate the
-							account.
-							{!isAdmin && 'Please contact your administrator for further help'}
-						</Typography.Paragraph>
-
-						<div className="cta">
+	const renderCustomerStories = (
+		filterCondition: (index: number) => boolean,
+	): JSX.Element[] =>
+		customerStoriesData
+			.filter((_, index) => filterCondition(index))
+			.map((story) => (
+				<CustomerStoryCard
+					avatar={story.avatar}
+					personName={story.personName}
+					role={story.role}
+					message={story.message}
+					link={story.link}
+					key={story.key}
+				/>
+			));
+
+	const tabItems: TabsProps['items'] = [
+		{
+			key: '1',
+			label: 'Why choose Signoz',
+			children: (
+				<Row align="middle" justify="center">
+					<Col span={12}>
+						<Row gutter={[24, 48]}>
+							<Col span={24}>
+								<InfoBlocks items={infoData} />
+							</Col>
+							<Col span={24}>
+								<Space size="large" direction="vertical">
+									<Flex vertical>
+										<Typography.Title level={3}>
+											Enterprise-grade Observability
+										</Typography.Title>
+										<Typography>
+											Get access to observability at any scale with advanced security and
+											compliance.
+										</Typography>
+									</Flex>
+									<List
+										itemLayout="horizontal"
+										dataSource={enterpriseGradeValuesData}
+										renderItem={(item, index): React.ReactNode => (
+											<List.Item key={index}>
+												<List.Item.Meta avatar={<CircleArrowRight />} title={item.title} />
+											</List.Item>
+										)}
+									/>
+								</Space>
+							</Col>
+							{isAdmin && (
+								<Col span={24}>
+									<Button
+										type="primary"
+										shape="round"
+										size="middle"
+										loading={isLoading}
+										onClick={handleUpdateCreditCard}
+									>
+										Continue to Upgrade
+									</Button>
+								</Col>
+							)}
+						</Row>
+					</Col>
+				</Row>
+			),
+		},
+		{
+			key: '2',
+			label: 'Your are in good company',
+			children: (
+				<Row gutter={[24, 16]} justify="center">
+					{/* #FIXME: please suggest if there is any better way to loop in different columns to get the masonry layout */}
```

Comment:
@sharpshooter90 : We can use css grid to achieve masonry layout. We can connect to check this.

Reply:
sure I will try it out